src/main/java/bio/terra/workspace/service/job/JobMapKeys.java

```diff
@@ -8,6 +8,9 @@
   STATUS_CODE("status_code"),
   AUTH_USER_INFO("auth_user_info"),
   SUBJECT_ID("subjectId"),
+  // Path for results of an async operation, relative to domain. Should be prefixed with /, e.g.
```

Comment:
I think you mean that the string does not have a leading /, right? So that is why it is up to the referencer to do the prefixing?

Reply:
I had originally meant it **does** have a leading /. Taking a second look, that's pretty confusing for something called relative path, though. I think I need to rename this for clarity, but I'm not quite sure what.

Reply:
Maybe it is just a matter of clarifying the documentation where it is set and used.

Reply:
Removed this comment - per Dan's other comment I switched to use `Path.of()` which handles this